src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -18,13 +18,15 @@ internal abstract class SynthesizedParameterSymbolBase : ParameterSymbol
         private readonly int _ordinal;
         private readonly string _name;
         private readonly RefKind _refKind;
+        private readonly bool _isNullChecked;
 
         public SynthesizedParameterSymbolBase(
             MethodSymbol container,
             TypeWithAnnotations type,
             int ordinal,
             RefKind refKind,
-            string name = "")
+            string name = "",
+            bool isNullChecked = false)
```

Comment:
`name` and `isNullChecked` do not need default values - all callers pass values explicitly.